system_tests/system_tests.go

```diff
@@ -71,7 +72,7 @@ var _ = Describe("Operator", func() {
 
 		It("works", func() {
 			By("being able to create a test queue and publish a message", func() {
-				response, err := rabbitmqAlivenessTest(hostname, port, username, password)
+				response, err := alivenessTest(hostname, port, username, password)
```

Comment:
Should we rename this to livenessTest to be consistent with the k8s terminology?

Reply:
aliveness test refers to rabbitmq aliveness test via management api `/api/aliveness-test/`. I think the name is fine here.